src/borg/item.pyx

```diff
@@ -656,6 +656,9 @@ class ItemDiff:
             changes.append(self._owner_diff())
             changes.append(self._mode_diff())
 
+        if not content_only:
+            changes.extend(self._time_diffs())
+
```

Comment:
Guess this need a clarification about what this feature should do exactly.

The way you implemented it is **without comparing timestamps**.

But the feature name "content-only" (== "data-only") rather suggests **without comparing metadata**.
See the lines above which compare owner and mode...

Reply:
After this is decided, the help text needs adjustment.